### ISSUE TITLE
Publish new version with lastest changes

### ISSUE THREAD

**Issue by juan-leon** [fgrehm/vagrant-lxc#415]
vagrant-lxc 1.2.1 has some bugs, already fixed in master, that prevents me from using "private_network" configs in Ubuntu 16.04 based boxes.  Could be possible to publish a new version?

(I know that the fixes solve that issue because I installed them manually (aka copying files)).

I tried to make a gem (using instructions on the wiki:  `bundler install; bundle exec rake build`) and use

```
vagrant plugin install vagrant-lxc --plugin-source file:///my/path/vagrant-lxc-1.2.1.gem
```

and other variants.  I had no success.  In Ubuntu 12.04 the error I get is 

```
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

Could not fetch specs from file:/my/path/vagrant-lxc-1.2.1.gem/

Warning: this Gemfile contains multiple primary sources. Using `source` more than once without a block is a security risk, and may result in installing unexpected gems. To resolve this warning, use a block to indicate which gems should come from the secondary source.
```

I did an strace and I see that a file named `specs.4.8.gz` is looked for, but I do not have it and have zero clues about how to generate it.

In Ubuntu 16.04 I got a different error:

```
Installing the 'vagrant-lxc' plugin. This can take a few minutes...
/usr/lib/ruby/2.3.0/rubygems/specification.rb:946:in `all=': undefined method `group_by' for nil:NilClass (NoMethodError)
        from /usr/lib/ruby/vendor_ruby/vagrant/bundler.rb:275:in `with_isolated_gem'
        from /usr/lib/ruby/vendor_ruby/vagrant/bundler.rb:231:in `internal_install'
        from /usr/lib/ruby/vendor_ruby/vagrant/bundler.rb:102:in `install'
        from /usr/lib/ruby/vendor_ruby/vagrant/plugin/manager.rb:62:in `block in install_plugin'
        from /usr/lib/ruby/vendor_ruby/vagrant/plugin/manager.rb:72:in `install_plugin'
        from /usr/share/vagrant/plugins/commands/plugin/action/install_gem.rb:37:in `call'
        from /usr/lib/ruby/vendor_ruby/vagrant/action/warden.rb:34:in `call'
        from /usr/lib/ruby/vendor_ruby/vagrant/action/builder.rb:116:in `call'
        from /usr/lib/ruby/vendor_ruby/vagrant/action/runner.rb:66:in `block in run'
        from /usr/lib/ruby/vendor_ruby/vagrant/util/busy.rb:19:in `busy'
        from /usr/lib/ruby/vendor_ruby/vagrant/action/runner.rb:66:in `run'
        from /usr/share/vagrant/plugins/commands/plugin/command/base.rb:14:in `action'
        from /usr/share/vagrant/plugins/commands/plugin/command/install.rb:32:in `block in execute'
        from /usr/share/vagrant/plugins/commands/plugin/command/install.rb:31:in `each'
        from /usr/share/vagrant/plugins/commands/plugin/command/install.rb:31:in `execute'
        from /usr/share/vagrant/plugins/commands/plugin/command/root.rb:56:in `execute'
        from /usr/lib/ruby/vendor_ruby/vagrant/cli.rb:42:in `execute'
        from /usr/lib/ruby/vendor_ruby/vagrant/environment.rb:268:in `cli'
        from /usr/bin/vagrant:173:in `<main>'

```

Most likely my lack of knowledge of ruby/rake/bundler is in the way, but in that case I would appreciate clues or doc updates.

Thanks

**Comment by juan-leon** [comment]
Did more digging.  Issue in Ubuntu 16.04 was because of https://bugs.launchpad.net/ubuntu/+source/vagrant/+bug/1562696

Once manually patched, same problem as precise (cannot fetch specs, plus the multiple sources warning)

**Comment by kathiedart** [comment]
Is this ok now on 1.2.0?

**Comment by ccope** [comment]
I think this was a bug fixed in vagrant 1.8.3: https://github.com/mitchellh/vagrant/commit/ba77d4b533e449fa610531a7832b8fd1837eb9db
Please try upgrading and open a new issue if you still see problems.

**Comment by kathiedart** [comment]
Aha, 1.8.1 was the latest in the repositories which is why I was still having problems.
